schemas/devfile.json

```diff
@@ -176,21 +176,21 @@
               "locationType": {
                 "description": "Type of plugin location",
                 "enum": [
-                  "Registry",
-                  "Url"
+                  "RegistryEntry",
+                  "Uri"
                 ],
                 "type": "string"
               },
               "memoryLimit": {
                 "type": "string"
               },
-              "registry": {
-                "description": "Location in a plugin registry",
+              "registryEntry": {
```

Comment:
I'm not sure on `registryEntry` vs `registry`. It seems wordier without additional clarity.

Reply:
That would be further discussed in the context of the following issue: https://github.com/che-incubator/devworkspace-api/issues/13#issuecomment-590913590

Do you think that we could merge this PR and postpone the final choice to a further PR ?

Reply:
:+1: